.github/workflows/ci.yml

```diff
@@ -743,16 +655,14 @@ jobs:
         run: |
           cd tee-worker/docker
           docker compose -f litentry-parachain.build.yml build
-      
+
       - name: Integration Test ${{ matrix.test_name }}
-        if: needs.set-condition.outputs.run_tee_test == 'true'
```

Comment:
why need to remove this line?

Reply:
I moved to whole job instead of skipping only step. As it seems redundant to trigger job without running tests in it. By the way, was there specific case why we need the job to be completed even if we skip tests?